config/webpack.config.js

```diff
@@ -197,9 +197,16 @@ function generateHtmlFiles(buildPath, scaffoldAssets) {
     template = {},
     widgetType,
     widgetTemplate,
+    defineWebComponentsLocally,
   }) =>
     new HtmlPlugin({
-      chunks: ['polyfills', 'web-components', 'vendor', 'style', entryName],
+      chunks: [
+        'polyfills',
+        defineWebComponentsLocally ? null : 'web-components',
```

Comment:
This change should only affect local dev scaffolding. If an app's registry entry has `defineWebComponentsLocally` set to true, then we don't want to include the `web-components` entry file on the HTML page that webpack generates.